packages/crayons-core/src/index.html

```diff
@@ -12,7 +12,22 @@
     <script nomodule src="/build/crayons.js"></script>
   </head>
   <body>
+    <fw-button expand mode="freshsales" color="primary"
```

Comment:
do we need to use `mode` ?.

Kindly comment the components here as this will increase the initial bundle size of the script.

Reply:
Reverted the changes to index.html, the changes was there to showcase the demo.